CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Change Log
 
+## [1.2.0b2] - 2022-05-30
+
+### Docs
+
+- Updated plugin management commands ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- Added `--readme` flag to documentation ([#5357](https://github.com/python-poetry/poetry/pull/5357))
+- Added example for multiple maintainers ([#5661](https://github.com/python-poetry/poetry/pull/5661))
+- Updated documentation for issues #4800, #3709, #3573, #2211 and PR #2414 ([#5656](https://github.com/python-poetry/poetry/pull/5656))
+- Added `poetry.toml` note in configuration ([#5492](https://github.com/python-poetry/poetry/pull/5492)) 
+- CLI: Added `about`, `help`, `list`, `--full-path`, `--all` documentation ([#5664](https://github.com/python-poetry/poetry/pull/5664))
+
+### Fixed
+
+- Fixed `mypy` inspection issues ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- Fixed `poetry update --dry-run` to not modify `poetry.lock` ([#5718](https://github.com/python-poetry/poetry/pull/5718), [#3666](https://github.com/python-poetry/poetry/issues/3666), [#3766](https://github.com/python-poetry/poetry/issues/3766))
+- Fixed #5682 by not raising a 404 exception when using private repository ([#5698](https://github.com/python-poetry/poetry/pull/5698))
+- Fixed #5537 where export fails to resolve dependencies with more than one path ([#5688](https://github.com/python-poetry/poetry/pull/5688))
+- Fixed #2445 and #5301 where environment variables, `POETRY_CONFIG_DIR` and `POETRY_CACHE_DIR`, are respected (#[5672](https://github.com/python-poetry/poetry/pull/5672))
+- Fixed more `truthy-bool` mypy warnings ([#5667](https://github.com/python-poetry/poetry/pull/5667))
+- Fixed #3628 and #4702 by handling invalid distributions gracefully ([#5645](https://github.com/python-poetry/poetry/pull/5645))
+
+### Added
+
+- Added support for multiple-constraint direct origin dependencies with the same version ([#5715](https://github.com/python-poetry/poetry/pull/5715))
+- Added support for `bool` values for TLS certificate verification of repositories ([#5719](https://github.com/python-poetry/poetry/pull/5719)
+- Added PEP-610 metadata for editable builds ([#5703](https://github.com/python-poetry/poetry/pull/5703))
+- Added `virtualenvs.prompt` option to config ([#5606](https://github.com/python-poetry/poetry/pull/5606))
+- Added @radoering to list of maintainers 🎉 ([#5632](https://github.com/python-poetry/poetry/pull/5632))
```

Comment:
That actually added everyone from the core team as the `maintainers` list wasn't there before.

Reply:
ah yea I noticed that later as I was going down the commits list: https://github.com/python-poetry/poetry/pull/5725/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR55